dags/openshift_nightlies/releases/4.7/openstack/default/install.json

```diff
@@ -0,0 +1,26 @@
+{
+    "orchestration_user": "stack",
+    "public_net_name": "nova",
+    "osp_project_name": "shiftstack",
+    "openstack_create_flavors": "true",
+    "openstack_upload_image": "true",
+    "openshift_base_domain": "example.com",
```

Comment:
is this default usable? If there's a variable with no default that *needs* to be added in release-specific variables then set them to `""` to indicate that a user should add this. See https://github.com/cloud-bulldozer/airflow-kubernetes/blob/master/dags/openshift_nightlies/tasks/install/cloud/defaults.json

Reply:
fixed